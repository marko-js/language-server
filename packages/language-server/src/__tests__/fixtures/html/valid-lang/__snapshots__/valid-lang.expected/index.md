## Diagnostics
### Ln 1, Col 2
```marko
> 1 | <p lang="invalid">
    |  ^ Fix all of the following:
  Value of lang attribute not included in the list of valid languages
  2 |   This isn't a language
  3 | </p>
```

