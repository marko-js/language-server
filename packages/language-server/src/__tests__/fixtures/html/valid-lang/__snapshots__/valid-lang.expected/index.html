<p data-marko-node-id="0" lang="invalid">
  This isn't a language
</p>